app/src/main/java/jayp/bge/playboikotlin/zeb/fragments/hurricankotalina/EightBallJson.kt

```diff
@@ -0,0 +1,8 @@
+package jayp.bge.playboikotlin.zeb.fragments.hurricankotalina
+
+data class EightBallJson(
+    val demand: String,
+    val id: Int,
+    val message: String,
+    val you_will_live_to: Int
```

Comment:
literALLY fuck snake case you need to make this camel case : jus cause the field is snake doesnt mean it needs to be serialized to snake 🐸